### ISSUE TITLE
Dust rune Ground Item colouring glitch

### ISSUE THREAD

**Issue by troypressley1** [runelite/runelite#7952]
When a stack of 400 Dust runes appears at a value of 1,200 gp, the ground items plugin gives it the colour of an item with a value of 2,500 or more. It's not in my list of highlighted items either. On a stack of 800 Dust runes valued at 2,400 gp, the ground items plugin gives it the colour of a low value item, which only happens for items at 9,000+ gp. This is the only item this glitch has ever happened to.

So basically, 1,200gp of Dust runes shows as a 2,500+gp item, and a 2,400gp stack of Dust runes shows as a 9,000+ gp item from the ground items plugin, even when there's nothing else on the stack.

**Comment by deathbeam** [comment]
It takes HA if HA > GE, 400 * 12 = 4800, etc etc
